### DISCUSSION TITLE
월드 출시 리스트에 URL 활용이 궁금해요.

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1281] [5. Q&A]
이번에 월드 테스트를 하기 위해서 제페토 스튜디오에 로그인하면서 알게 된 것인데
유튜브 URL을 첨부할 수 있도록 해놨더군요.
혹시 유튜브 링크를 첨부하면 어떻게 되는지 알 수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 개발자의 소셜 계정 링크입니다.
자신의 계정 및 월드 홍보용으로 사용되거나 
플레이어들이 개발자와 소통하고 싶을때 해당 소셜 계정에 들어가서 연락을 하는데 사용될 수 있습니다.

감사합니다.

**Reply by zlmrzlo** [reply]
혹시 이 링크가 제페토앱에 노출되는 형태인가요?

**Reply by Zepeto-TechEvangelist** [reply]
네, 추후에 반영될 예정입니다. 그때 정식으로 관련 공지가 올라갈 예정입니다. 
현재는 스튜디오 쪽에서 먼저 검수를 위한 데이터 수집 중입니다.
미리 커뮤니티 및 유튜브 채널을 준비하시면 좋을 것 같습니다.

감사합니다.
